tests/framework/installer/ceph_manifests.go

```diff
@@ -661,6 +661,8 @@ spec:
   storage:
     useAllNodes: true
     useAllDevices: ` + strconv.FormatBool(settings.UseAllDevices) + `
+    directories:
+    - path: ` + settings.DataDirHostPath + /* simulate legacy fallback osd behavior so existing tests still work */ `
```

Comment:
If this dir is always added in the tests, a side effect may be that there are two OSDs in the tests where a device is consumed. I'm not sure it will cause issues, just something to be aware of.

Reply:
I considered that as well. I'm not sure how to detect whether the environment has disks and then use that (along with the UseAllDevices setting) to determine if the dir should be added or not. This is definitely close to the smallest change that could be made to work. I don't think there will be an issue with there being more than one OSD in tests, and so I'm not sure how useful it would be to spend time trying to keep to a one-disk-only environment. To be sure, I'd welcome more thoughts here.